frappe/tests/test_commands.py

```diff
@@ -691,6 +691,23 @@ def test_migrate_cli(self):
 			self.assertEqual(result.exception, None)
 
 
+class TestAddNewUser(BaseTestCommands):
+	def test_create_user(self):
+        self.execute(
```

Comment:
This uses spaces for indentation, we use tabs everywhere 😅 

This is causing failure. In CI. We can't edit this PR because it's raised from the organization's account and not a user account.

Reply:
I fix it can you check please